winml/lib/Api.Ort/OnnxruntimeEngine.cpp

```diff
@@ -1189,7 +1187,7 @@ HRESULT OnnxruntimeEngineFactory::RuntimeClassInitialize() {
   return S_OK;
 }
 
-STDMETHODIMP OnnxruntimeEngineFactory::EnsureEnvironment() {
```

Comment:
What is STDMETHODIMP resolve to again?

Reply:
it gives you __stdcall, which doesnt match this implementation's decl

---
In reply to: [372722706](https://github.com/microsoft/onnxruntime/pull/2943#discussion_r372722706) [](ancestors = 372722706)